README.md

```diff
@@ -185,3 +185,13 @@ String. A space-separated list of NTP servers to use for time synchronization. D
 ### dnsServers
 
 String. A space-separated list of preferred DNS servers to use for name resolution. Falls back to DHCP provided servers and Google DNS.
+
+## Yocto version support
+
+The following Yocto versions are supported:
+ * Morty (2.2)
+  * **TESTED**
+ * Krogoth (2.1)
+  * **TESTED**
+ * Jethro (2.0)
+  * **UNTESTED**
```

Comment:
Sorry, I got the versions wrong, I've pushed a fix. The agreed approach in the arch call (which I missed as I had to leave early on Monday to do a meetup) was to keep the most recent version that we don't build/test.